diff -Naur gpac-2bac32cd96c54fbcd9f88c5d6a3c2965738efd28.old/Makefile gpac-2bac32cd96c54fbcd9f88c5d6a3c2965738efd28/Makefile
--- gpac-2bac32cd96c54fbcd9f88c5d6a3c2965738efd28.old/Makefile	2018-09-26 15:57:22.536073234 +0200
+++ gpac-2bac32cd96c54fbcd9f88c5d6a3c2965738efd28/Makefile	2018-09-26 15:57:37.391337760 +0200
@@ -140,9 +140,11 @@
 endif
 	$(INSTALL) -d "$(DESTDIR)$(mandir)"
 	$(INSTALL) -d "$(DESTDIR)$(mandir)/man1"
-	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/mp4box.1 $(DESTDIR)$(mandir)/man1/ 
-	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/mp4client.1 $(DESTDIR)$(mandir)/man1/ 
-	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/gpac.1 $(DESTDIR)$(mandir)/man1/
+	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/dashcast.1 $(DESTDIR)$(mandir)/man1/DashCast.1
+	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/mp4box.1 $(DESTDIR)$(mandir)/man1/MP4Box.1
+	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/mp4client.1 $(DESTDIR)$(mandir)/man1/MP4Client.1
+	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/mp42ts.1 $(DESTDIR)$(mandir)/man1/MP42TS.1
+	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/man/gpac.1 $(DESTDIR)$(mandir)/man1/gpac.1
 	$(INSTALL) -d "$(DESTDIR)$(prefix)/share/gpac"
 	$(INSTALL) $(INSTFLAGS) -m 644 $(SRC_PATH)/doc/gpac.mp4 $(DESTDIR)$(prefix)/share/gpac/  
 	$(INSTALL) -d "$(DESTDIR)$(prefix)/share/gpac/gui"
@@ -215,8 +217,10 @@
 	rm -rf $(DESTDIR)$(prefix)/bin/MP4Client
 	rm -rf $(DESTDIR)$(prefix)/bin/MP42TS
 	rm -rf $(DESTDIR)$(prefix)/bin/DashCast
-	rm -rf $(DESTDIR)$(mandir)/man1/mp4box.1
-	rm -rf $(DESTDIR)$(mandir)/man1/mp4client.1
+	rm -rf $(DESTDIR)$(mandir)/man1/MP4Box.1
+	rm -rf $(DESTDIR)$(mandir)/man1/MP4Client.1
+	rm -rf $(DESTDIR)$(mandir)/man1/MP42TS.1
+	rm -rf $(DESTDIR)$(mandir)/man1/DashCast.1
 	rm -rf $(DESTDIR)$(mandir)/man1/gpac.1
 	rm -rf $(DESTDIR)$(prefix)/share/gpac
 	rm -rf $(DESTDIR)$(prefix)/include/gpac
